src/main/java/dev/aurelium/auramobs/entities/AureliumMob.java

```diff
@@ -51,6 +51,28 @@ public AureliumMob(LivingEntity mob, int level, AuraMobs plugin) {
         Expression resHealth = new ExpressionBuilder(healthFormula).build();
         double damage = BigDecimal.valueOf(resDamage.evaluate()).setScale(2, RoundingMode.CEILING).doubleValue();
         double health = resHealth.evaluate();
+
+        if(plugin.optionString(prefix + "damage.max") != null && !plugin.optionString(prefix + "damage.max").isEmpty()){
```

Comment:
Add a space between if and open paren, and between closing paren and curly bracket